src/examples/advection/advection_example.jl

```diff
@@ -1,17 +1,25 @@
-# advection test
+# Advection Example
```

Comment:
The directory structure makes total sense, but I would actually move the examples folder to the top directory instead of having it in `src`, since this "miniapp" is not part of the Gridtools package.